source/ocsp-support/ocsp-support.rst

```diff
@@ -102,8 +102,10 @@ invalid, the driver SHOULD end the connection.
     attempt to validate the status of the unvalidated certificates
     using the cached CRLs.
 
-7.  If the server’s certificate remains unvalidated and that certificate
-    has an OCSP endpoint, the driver SHOULD reach out to the OCSP
+7.  If the server’s certificate remains unvalidated, that certificate
+    has an OCSP endpoint and ``tlsDisableOcspEndpointCheck`` is false
```

Comment:
I think Ocsp should be all caps (tlsDisableOCSPEndpointCheck) similar to tlsCAFile.

Reply:
Ah, sorry, I was instinctively following the camelCasing naming convention (which .NET/C# also uses) in which acronyms longer than two letters only have their initial letter capitalized, which can help with readability in the case where multiple acronyms occur together.  The only other multi letter acronym we have in the URI options appears to be "tls", but that occurs at the beginning of the word, although having it be lower case also fits with the aforementioned naming convention. 

Playing devil's advocate, the server team appears to use all caps for their mongod parameter options e.g. CRLFile (https://docs.mongodb.com/manual/tutorial/configure-ssl/)

See also: https://en.wikipedia.org/wiki/Camel_case#Programming_and_coding and https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions

Give the above I'd prefer to keep the name as is, but  if you feel strongly about naming capitalizing OCSP, I'm fine with changing it :)

Reply:
I think the (very slight) precedent is for all caps from tlsCAFile like Bernie mentioned, with `tls` being lowercase because I don't think anybody would prefer something like `TLSCAFile` or `tLSCAFile`. That being said, it shouldn't actually matter much since the URI options are required to be case-insensitive. How about we use the case `OCSP` in the spec, but allow languages to use whatever casing their ecosystem prefers?

Reply:
Ah, I totally forgot that URI options are case insensitive. In which case, I'll just change it to `OCSP`